README.md

```diff
@@ -8,6 +8,11 @@ Android app for watching videos.
 
 If you want to make changes to the elimu.ai Content Provider's utility library, see testing instructions at https://github.com/elimu-ai/content-provider/blob/main/README.md#utils-snapshot
 
+### How to release a new version?
+
+1. Update versionCode, versionName in `app/build.gradle` file by increment either major/minor/patch number and merge to `main` (Do not remove the `SNAPSHOT` suffix)
```

Comment:
> (Do not remove the `SNAPSHOT` suffix)

@tuancoltech Normally the manual procedure would be to remove `-SNAPSHOT` and then perform a release. So can you explain why this instruction is opposite?

Reply:
@jo-elimu Well, actually it doesn't matter the `SNAPSHOT` prefix is removed or not, the release script should work anyway. So, I've updated the guide.